tests/cli/test_config.py

```diff
@@ -80,16 +80,10 @@ def test_config_profile():
     list_profiles(configuration, print_yaml=True)
 
 @pytest.mark.cli
-def test_config_profile_remove():
+def remove_profiles(configuration, names):
```

Comment:
This is a regression test method it needs to start with `test_` so name it `test_profiles_remove`. 

Remove the arguments it should simply be 

```python
def test_profiles_remove()
```